### ISSUE TITLE
tool tips?

### ISSUE THREAD

**Issue by allisonmuise** [Esri/crowdsource-manager#20]
tool tips seem to be missing from the UI. Please include tootips for all buttons and wherever else they may be needed for screenreaders
